packages/terra-framework-docs/src/terra-dev-site/doc/menu/example/BasicMenu.module.scss

```diff
@@ -1,5 +1,11 @@
 :local {
   .menu-wrapper {
     display: inline-block;
+
+    button {
+      &:focus {
+        box-shadow: 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35);
```

Comment:
Please add the equivalent styles for clinical and fusion themes

Reply:
Added focus styles for clinical and fusion themes